### PR TITLE
DM-31465: rename lsst.ts.MTMount to lsst.ts.mtmount

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ Please only do this if you are sure it will not interfere with operations:
```
Comment (wvreeven) [inline doc/index.rst:41]: This is about line 34. I kindly suggest to replace `When MTMount is started` with `When the MTMount CSC is started`.

File: doc/version_history.rst [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
 Version History
 ###############
 
+v0.21.0
+-------
+
+Changes:
+
+* Renamed the package from ts_MTMount to ts_mtmount,
+  and the Python namespace from lsst.ts.MTMount to lsst.ts.mtmount.
```
Comment (wvreeven) [inline doc/version_history.rst:15]: A bit picky but shouldn't one line be used per sentence instead of splitting it out over two?

**Review by wvreeven — APPROVED** [review]
The changes look good to me. I have two minor comments about documentation that I leave for you to consider.

File: doc/version_history.rst (hunk 2 above)
Comment (r-owen) [reply doc/version_history.rst:15]: The line was long enough that I chose to break it at the comma. I often do that. Technically perhaps I shouldn't.

**Pull request merged**